test/smokes/pmd_java/expectations.rb

```diff
@@ -7,20 +7,9 @@
   type: "success",
   analyzer: { name: "PMD Java", version: default_version },
   issues: [
-    {
-      message: "Do not add empty strings",
-      links: %W[https://pmd.github.io/pmd-#{default_version}/pmd_rules_java_performance.html#addemptystring],
-      id: "AddEmptyString",
-      path: "src/Hello.java",
-      location: { start_line: 17, start_column: 16, end_line: 17, end_column: 17 },
-      object: { ruleset: "Performance", priority: "3" },
-      git_blame_info: {
-        commit: :_, line_hash: "846dcdb29796bc0ef3e1c3b6c6ccd7ccd6c352f7", original_line: 17, final_line: 17
-      }
-    },
     {
       message: "Avoid calling finalize() explicitly",
-      links: %W[https://pmd.github.io/pmd-#{default_version}/pmd_rules_java_errorprone.html#avoidcallingfinalize],
+      links: %W[https://pmd.github.io/pmd-6.33.0/pmd_rules_java_errorprone.html#avoidcallingfinalize],
```

Comment:
```suggestion
      links: %W[https://pmd.github.io/pmd-#{default_version}/pmd_rules_java_errorprone.html#avoidcallingfinalize],
```

Revert the code, then this test will pass.

Reply:
OK, ready.